### PR TITLE
my changes

### PR DESCRIPTION
**Pull request by nielsarends**

### CODE REVIEW

File: pietrasanta.food.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      <h2>Gastromonics</h2>
+
+        <p>By all means this is conviviality and joy. And slowness of course. Enjoying a meal with friends or family is the main ingredient of any Italian dish. Italian Sunday lunches are the best example, a unique experience you should try at least once. Italian sunday lunch They say only 5% of Italians eat out on Sundays, while 95% are at home eating as much as humanly possible. Sunday lunch is usually spent at some relatives’ home, in my case my grandparents’.
+
+          In any household there is always a grandma or aunt who thinks you are malnourished, too skinny and need to be filled as a Christmas turkey. You will not be able to leave your grandma’s table if you do not eat every single crumble she thinks you need.
+
+          This will also be your fate if you are invited for lunch at an Italian gathering... be prepared! </p>
+
+          <div class="smallpic" id="pig">
+          <img src=assets/images/pig.jpg>
```
Comment (mtimofiiv) [inline pietrasanta.food.html]: your image tag does not have the url of the image in quotations

File: assets/style.css [hunk 2]
```diff
@@ -36,3 +36,45 @@
...
+
+.buttons{
+  margin: 40px;
+  text-align: center;
+}
+
+.smallpic{
+  width: 80%;
+  margin: auto;
+  padding: 20px
```
Comment (biancabesciu) [inline assets/style.css]: you forgot to close the semicolons for the padding

File: pietrasanta.sports.html [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+      <h2>Biking</h2>
+
+        <p>Of all European countries, none draws visitors over and over again like Italy. A peninsula extending into the Mediterranean Sea, Italy consists of 21 diverse regions of rolling hills, vineyards, mountains, lakes, cliffs and islands.
+
+            Each region has its unique charm, but all share the ‘Italian’ view that life should be taken at a slow pace with time to enjoy conversation, food and drink. Together, Italy’s regions boast the highest number of UNESCO World Heritage Sites in the world.
+
+              Rome, Florence, Venice, Assisi and Pisa are a few of the country's renowned cities along with Italy’s 95,000 monumental churches, 40,000 forts and castles and 5,600 museums and archaeological sites. Whether you ride a gondola through Venice’s Grand Canal, stroll along Lake Como or visit the Coliseum in Rome, the opportunities are endless when visiting Italy. </p>
+
+          <div class="smallpic">
+          <img src=assets/images/cipo.jpg>
```
Comment (biancabesciu) [inline pietrasanta.sports.html]: img tag is missing the quotations

File: pietrasanta.sports.html [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+          </div>
+
+          <p> Mario Cipollini; born 22 March 1967), actually lives in Pietrasanta. The legend is often abbreviated to "Cipo", is a retired Italian professional road cyclist most noted for his sprinting ability, the longevity of his dominance (his first pro win came in 1989, his last in 2005; 191 victories in all) and his colourful personality. His nicknames include Il Re Leone (The Lion King) and Super Mario. He is regarded as one of the best sprinters of his generation. </p>
+
+          <div class="smallpic">
+          <img src=assets/images/berg.jpg>
```
Comment (biancabesciu) [inline pietrasanta.sports.html]: img tag is missing the quotations

File: pietrasanta.food.html (hunk 1 above)
Comment (nielsarends) [inline pietrasanta.food.html]: fixed it.

File: assets/style.css (hunk 2 above)
Comment (nielsarends) [inline assets/style.css]: fixed. tks.

File: pietrasanta.sports.html (hunk 3 above)
Comment (nielsarends) [inline pietrasanta.sports.html]: fixed. tks.

File: pietrasanta.sports.html (hunk 4 above)
Comment (nielsarends) [inline pietrasanta.sports.html]: fixed. tks.

**Pull request merged**